torch/autograd/gradcheck.py

```diff
@@ -9,6 +9,7 @@
 from torch._vmap_internals import vmap, _vmap
 import functools
 
+__all__ = ["gradcheck", "gradgradcheck"]
```

Comment:
I just added the documented API to this list. should we document these other functions if they are considered public?

Reply:
Since we're just keeping them for BC reasons, maybe it's okay if there are not? The only reason to add them to `__all__` would be so that if someone used to do `from autograd.gradcheck import *` it wouldn't break them.

Reply:
Yeah we probably want to document GradcheckError though

Reply:
For GradcheckError, we can probably replace that comment with the following:
```
r"""Error raised by :func:`gradcheck` and :func:`gradgradcheck`
```
I don't know if we want to be more specific and say "... when the gradients are incorrect" because we also sometimes raise GradcheckError when the inputs are invalid (but we also raise ValueError sometimes for those cases as well... for BC reasons?).

Reply:
updated!